# 4.16 Age at First Marriage

In [1]:
import math
""" Confidence Interval of 95%  Function

pe = point estimate
std = standard deviation of the population
sample_size = size of the sample

"""
#Standard error method that comes into computation
def standard_error(std, sample_size):
    return (std/(math.sqrt(sample_size)))


def ci(mean, std, sample_size,conf_lvl):
    z = {80:1.282,85:1.440,90:1.645,95:1.960,99:2.576,99.5:2.807,99.9:3.291}
    #needs the standard_error
    se = standard_error(std,sample_size)
    ci_minus = mean - (z[conf_lvl] * se)
    ci_plus = mean + (z[conf_lvl] * se)
    ci = (ci_minus,ci_plus)
    #returns a tuple
    return ci

**Given Data :** 
<br/>
Sample size = 5534<br/>
Average age at first marriage = 23.44 <br/>
Standard Deviation = 4.72

Creating a 95 % confidence interval for the average age at first marriage of women.

In [2]:
ci(23.44,4.72,5534,95)

(23.315640635918452, 23.56435936408155)

**<u>Inference : </u> : ** We are 95% confident that the age of first marriage for women is between 23.315 and 23.56